crates/pop-cli/src/commands/call/contract.rs

```diff
@@ -901,7 +910,26 @@ mod tests {
 			dry_run: false,
 			execute: false,
 			dev_mode: false,
-		}.configure(&mut cli, false).await, Err(message) if message.to_string().contains("Unable to fetch contract metadata")));
+		};
+		let mut cli = MockCli::new();
+		assert!(
```

Comment:
For future reference, you could have put all the variants into a vec/array and looped over it to avoid all this duplicate code.